<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-OL3DFRR" failures="1" tests="2" name="dataProvider.ToLearnDataProvider1" time="9.591" errors="0" timestamp="2024-11-04T22:58:26 IST" skipped="0">
  <testcase classname="dataProvider.ToLearnDataProvider1" name="login" time="4.354">
    <failure message="UserFailed to Login expected [SkillRary Ecommerce] but found [SkillRary Courses]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: UserFailed to Login expected [SkillRary Ecommerce] but found [SkillRary Courses]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at dataProvider.ToLearnDataProvider1.login(ToLearnDataProvider1.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
    </failure>
  </testcase> <!-- login -->
  <system-out/>
  <testcase classname="dataProvider.ToLearnDataProvider1" name="login" time="5.237"/>
  <system-out>
    <![CDATA[User Logged in Successfully!]]>
  </system-out>
</testsuite> <!-- dataProvider.ToLearnDataProvider1 -->
